docs/user/use_pretrained_network_to_start_new_experiment.md

```diff
@@ -0,0 +1,79 @@
+# Use a pre-trained network to start a new experiment
+
+In this section you will learn how to load a pre-trained network and use it in
+new experiments. We'll cover two cases in particular:
+
+- How to use a trained policy as an expert in Behavioral Cloning
+- How to reuse a trained Q function in DQN
+
+Before attempting either of these, you'll need a saved experiment snapshot. [This page](save_load_resume_exp.md)
```

Comment:
Maybe link to the actual garage doc (like [This page](https://garage.readthedocs.io/en/latest/user/save_load_resume_exp.html) )makes more sense